src/modules/aws.rs

```diff
@@ -118,6 +118,49 @@ fn alias_region(region: String, aliases: &HashMap<String, &str>) -> String {
     }
 }
 
+fn get_credential_process(context: &Context, aws_profile: Option<&Profile>) -> Option<String> {
+    if let Some(cred_proc) = context.get_env("credential_process") {
```

Comment:
From the [AWS docs](https://docs.aws.amazon.com/sdkref/latest/guide/setting-global-credential_process.html) it doesn't look like you can set this can be read from an env var.

Reply:
I added this option because almost all tests in the module would have to be rewritten in order to create the temporary config file. I thought that option would be quite clunky as opposed to just checking for an env var. I also couldn't find an example of another program who uses a `credential_process` env var that could conflict with this.
With that being said, I'd appreciate some advice in this case. Should I stick with the env var even if it is not technically supported for the sake making the tests more readable and concise or should I rewrite all of them? Is there another way around this?
Thanks in advance.

Reply:
Since you have also added support for the `AWS_...` env vars you could use that as an alternative.